docs/data_management/tutorial/globus_group_tutorial.md

```diff
@@ -0,0 +1,399 @@
+
+1. When the installation is complete, click the "Finish" button to complete the GCP installation.
+
+    ![Completing GCP Setup highlighting Finish button.](./images/gg-gcp-install/win/005-finish.png)
+
+1. Following installation, GCP will launch in a new window. If it does not, look for it in your Start Menu.
+
+    When GCP has started, click on "Log In" to authenticate with Globus to begin the Collection setup process. This is a one-time setup to configure GCP to allow your machine to act as a Collection, enabling research data transfer with your computer.
+
+    ![GCP Setup window with Log In button.](./images/gg-gcp-install/win/006-authenticate.png)
+
+    Note that if you uninstall and reinstall GCP, you will need to complete this process again. You should not need to repeat this process otherwise.
+
+1. Grant the required consents. This is required to set up your computer as a Collection. Also provide a name for your consents. We recommend choosing a name that is short, memorable, and related to the purpose for the Collection.
+
+    ![GCP Setup form with required consents and Collection name field.](./images/gg-gcp-install/common/consents.png)
+
+1. Enter the details for your GCP Collection, and click save to continue. The following list describes the fields in the form shown below.
+
+    - **Owner Identity:** is the person responsible for this Collection. This field should already be filled with UAB Campus or XIAS email address. If not, please that email address here.
+    - **Collection Name:** is the name for the Collection. This should be filled with the name of the Collection from the previous step.
+    - **Description:** Feel free to enter descriptive information about the Collection here. This information will be displayed in the Globus Web App when the Collection is viewed by others.
+    - **High Assurance:** Only check this box if the Collection has or will have PHI, HIPAA, or other protected data. If this is the case, please ensure that you have already completed a risk assessment with UAB Enterprise IT.
+
+    ![GCP Setup Collection details](./images/gg-gcp-install/common/collection-details.png)
+
+1. GCP Setup is now complete on your computer. Your computer is now serving Globus Collection and may be used to transfer data. Click "Exit Setup" to close the window.
+
+1. After installation, you should see a lowercase letter "g" in a circle in your Windows system tray, typically at the bottom-right of the display. If you do not, try finding the Globus Connect Personal application in your start menu and starting the application.
+
+    ![GCP Icon in Windows system tray.](./images/gg-gcp-install/win/009-system-tray-icon.png)
+
+Continue on with [How Do I Choose Specific Folders Using Globus Connect Personal?](#how-do-i-choose-specific-folders-to-share-using-globus-connect-personal).
```

Comment:
The question mark should be removed here as well.

Reply:
Same here.